types/jsonwebtoken/index.d.ts

```diff
@@ -1,9 +1,10 @@
-// Type definitions for jsonwebtoken 7.2.2
+// Type definitions for jsonwebtoken 7.2.3
```

Comment:
Looks like our lint rules are disabled for this package. Can you enable the lints in our `tslint.json` and fix up the version string to only be a `MAJOR.MINOR` version (e.g. `4.7`), as opposed to including the patch or containing a leading `v` (e.g. `v1.2.3`).

Reply:
I've updated the comment to show the latest version of the `jsonwebtoken` package. Is this the correct thing to do or should it be targeting the first version that the typings are valid for @DanielRosenwasser? Also I'm not sure what you mean that lint rules are disabled for this package. Can you elaborate on that and I'll take another look.

Reply:
Alright I'll have a look. Thanks for that.

Reply:
I've removed those rules and fixed all the linting issues in my latest commit @DanielRosenwasser. Thanks again for looking into this PR.